### PR TITLE
Fix docs smoke error

### PR DESCRIPTION
**Pull request by bluwy**
## Changes

Add `@astrojs/sitemap` to `ssr.external` when loading config. It should be simplified when https://github.com/vitejs/vite/pull/10939 lands.

## Testing

Astro smoke test should pass.

## Docs

n/a bug fix

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "format:imports": "organize-imports-cli ./packages/*/tsconfig.json ./packages/*/*/tsconfig.json",
     "test": "turbo run test --output-logs=new-only --concurrency=1 --filter=astro --filter=create-astro --filter=\"@astrojs/*\"",
     "test:match": "cd packages/astro && pnpm run test:match",
-    "test:smoke": "turbo run build --filter=\"@example/*\" --filter=\"astro.build\" --filter=\"docs\" --output-logs=new-only --concurrency=1",
+    "test:smoke": "turbo run build --filter=\"@example/*\" --filter=\"docs\" --output-logs=new-only --concurrency=1",
```
Comment (bluwy) [inline package.json:21]: There doesn't look like a `astro.build` package to test so removed here

**Review by bholmesdev — APPROVED** [review]
Smost excellent fix of 2023 👏

File: package.json (hunk 1 above)
Comment (bholmesdev) [reply package.json:21]: Hm, there _was_ an `astro.build` package but the checkout code was removed. We should add that back in a separate PR

**Comment by aFuzzyBear** [comment]
> Smost excellent fix of 2023 👏

... so far 😉

**Pull request merged**